deployment/hosts

```diff
@@ -1,8 +1,7 @@
 upstream ansible_host=<upstream-host> ansible_user=<upstream-user>
 
 [hyperfoil_controller]
-<injector-controller-host> ansible_hostname=<injector-controller-host> ansible_host=<injector-controller-host> ansible_user=root
```

Comment:
Tempting it might be to remove `ansible_hostname` but I suspect errors to be thrown by Ansible scripts [here](https://github.com/Hyperfoil/hyperfoil_setup/blob/master/tasks/main.yml#L130), [here](https://github.com/Hyperfoil/hyperfoil_setup/blob/master/tasks/main.yml#L134), [here](https://github.com/Hyperfoil/hyperfoil_setup/blob/master/tasks/main.yml#L142), [here](https://github.com/Hyperfoil/hyperfoil_setup/blob/master/tasks/main.yml#L144), [here](https://github.com/Hyperfoil/hyperfoil_setup/blob/master/tasks/main.yml#L146), [here](https://github.com/Hyperfoil/hyperfoil_setup/blob/master/tasks/main.yml#L151), [here](https://github.com/Hyperfoil/hyperfoil_test/blob/master/tasks/main.yml#L6), [here](https://github.com/Hyperfoil/hyperfoil_test/blob/master/tasks/main.yml#L12), [here](https://github.com/Hyperfoil/hyperfoil_shutdown/blob/master/tasks/main.yml#L2) and [here](https://github.com/Hyperfoil/hyperfoil_shutdown/blob/master/tasks/main.yml#L8).

Reply:
I checked with 

```
[hyperfoil_controller]                                                  
192.168.58.3.nip.io ansible_host=192.168.58.3.nip.io ansible_user=eguzki
                                                                        
[hyperfoil_agent]                                                       
192.168.58.3.nip.io ansible_host=192.168.58.3.nip.io ansible_user=eguzki
```

And worked like a charm :)

Reply:
Before starting a run did you kill off the HF controller and re-run  (profiled-)injector.yaml ?

Reply:
I tested with fresh new controller and worked well.

Reply:
It is failing in your env?

Reply:
I recall on 2 occasions in the past "cleaning up" the same host variable. Causing the hyperfoil role to not execute. Can't explain why this is working for you now.